.github/workflows/default.yml

```diff
@@ -0,0 +1,121 @@
+name: default
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
```

Comment:
Let's add explicit quotes all occurrences of `python-version`, like `python-version: "3.9"` to avoid future conflicts of Python version 3.10 and YAML's floating point number interpretation.